app/messaging/email/create-submission-msg.js

```diff
@@ -190,9 +258,16 @@ function getSpreadsheetDetails(submission, desirabilityScore) {
   console.log('GET SPREADSHEET DETAILS')
   const today = new Date()
   const todayStr = today.toLocaleDateString('en-GB')
-  // const schemeName = 'Laying Hens for Health and Welfare'
   const subScheme = 'FTF-AHW-Laying Hens'
-  const subTheme = 'Laying Hens for health and welfare'
+  const subTheme = 'Laying hens'
+
+  const DD = String(today.getDate()).padStart(2, '0')
+  const MM = String(today.getMonth() + 1).padStart(2, '0')
+  const YYYY = today.getFullYear()
+  const hh = String(today.getHours()).padStart(2, '0')
+  const mm = String(today.getMinutes()).padStart(2, '0')
+
+  submission.dateTimeToday = `${DD}/${MM}/${YYYY} ${hh}:${mm}`
```

Comment:
you can achieve all this with one line 
new Date().toLocaleString('en-GB', { timeZone: 'Europe/London' })

Reply:
The format we need is DD/MM/YYYY hh:mm

I thought toLocaleString would do everything, including seconds, which we do not want?